src/components/templates/refactored/menu/webMenu/webMenu.tsx

```diff
@@ -21,21 +21,22 @@ const WebMenu = () => {
   const [searchValue, setSearchValue] = useState('');
 
   function navigateToSearch(q: string) {
-    navigate({
-      to: '/d/search',
-      search: (p: PayloadModel) => {
-        const type = p.type ?? 'projects';
-        const page = p.page ?? 1;
-        return { type, q, page };
-      },
-    });
+    navigate(
+      '/d/search',
+      //  ,
```

Comment:
Is this not reqiured anymore?

Reply:
It might be required, but since 'search' is not refactored yet I preferred to keep it here for later decisions